docs/signals/configuration/views-services/index.md

```diff
@@ -1,21 +1,65 @@
 ---
-title: "Views and Services"
+title: "Attribute groups: views and services"
 sidebar_position: 30
-sidebar_label: "Views and Services"
+sidebar_label: "Attribute groups"
 ---
 
-To access attributes, you need to define a `View` and optionally group multiple views into a `Service`. This guide explains how to define views and services, apply them to your deployment, and retrieve calculated values.
+Signals has two attribute groupings:
+* Views, for defining attributes
+* Services, for consuming attributes
 
+Each view is a versioned collection of attributes, and specific to one entity and one data source (stream or batch).
 
-### What is a view?
-A `View` is a versioned collection of attributes grouped by a common `Entity` (e.g., `session_id` or `user_id`). Once defined, a `View` allows you to retrieve the calculated values of the attributes it contains.
+Services are groups of views. They provide a stable interface to use in your applications: by pinning specific view versions, they provide a consistent set of consumable attributes. Services can contain views from different entities, and different sources.
 
-### What is a service?
-A `Service` is a collection of views that streamlines the retrieval of multiple views. By grouping related views into a `Service`, you can efficiently manage and access user insights, making it easier to personalize applications and analyze behavior.
+By using services you can:
+* Iterate on attribute definitions without worrying about breaking downstream processes
+* Migrate to new view versions by updating the service definition, without having to update the application code
 
+## Attribute management
 
-### Defining a view
-You can define a `View` by passing in a list of previously defined attributes. Here's an example:
+TODO this is very confusing
```

Comment:
Agree that this is hard to understand, I think we should separate the different cases and not try to explain "views with attributes" (those that are calculated in the stream or batch engine) and "views based on existing customer tables" (those where the user defines their own table and then provides a data source for it). The data sources are only necessary to define in the latter case, in the former (which is what Signals is really about), data sources should be handled automatically.